AppDelegate+Injection.swift

```diff
@@ -111,23 +111,27 @@ extension Resolver: ResolverRegistering {
         .scope(.application)
         main.register { OverridingFeatureFlagProvider() }.scope(.cached)
         main.register { DefaultFeatureFlagProvider() }.scope(.cached)
+        main.register { DeviceFeatureFlagProvider() }.scope(.cached)
 #if !DEBUG
         main.register { FirebaseFeatureFlagProvider() }.scope(.cached)
 #endif
         main.register {
 #if DEBUG
             CompositeFeatureFlagProvider(children: [
                 Resolver.resolve(OverridingFeatureFlagProvider.self),
+//                Resolver.resolve(DeviceFeatureFlagProvider.self),
```

Comment:
Why is it commented out?

Reply:
To be sure, it won't work unless the whole feature will be implemented.